R/lf_inflator.R

```diff
@@ -15,7 +15,7 @@
 #' If the SDMX connection fails, a message is emitted (not a warning) and
 #' the function contines as if \code{useABSConnection = FALSE}.
 #' 
-#' The internal data was updated on 2019-10-29 to 2019-09-01.
+#' The internal data was updated on 2019-11-15 to 2019-11-01.
```

Comment:
Should this be "updated to 2019-10-01"? That's the latest LFS release (released 2019-11-14)